src/steps/extract-metadata.js

```diff
@@ -180,14 +182,17 @@ export default function extractMetaData(state, req) {
   if (Object.keys(metaConfig).length > 0) {
     // add rest to meta.custom
     meta.custom = Object.entries(metaConfig)
-      .filter(([name]) => !ALLOWED_RESPONSE_HEADERS.includes(name))
+      .filter(([name]) => !IGNORED_CUSTOM_META.includes(name))
       .map(([name, value]) => ({
         name,
         value,
-        property: name.includes(':'),
+        property: name.includes(':') && !name.startsWith('twitter:'),
```

Comment:
you don't need this anymore neither :-)

```suggestion
        property: name.includes(':'),
```